src/lib.rs

```diff
@@ -156,6 +159,13 @@ compile_error!("A least one underlying storage type must be enabled. See the fea
 #[doc(hidden)]
 pub extern crate num;
 
+#[doc(hidden)]
+#[cfg(feature = "serde")]
+pub extern crate serde;
+
+#[cfg(all(feature = "serde", test))]
```

Comment:
Change order so that `test` appears first. `#[cfg(all(test, feature = "serde"))]`. I also have the dev-dependencies in a separate section beneath the regular dependencies. Not sure if this should be maintained or if all dependencies should just be included in order. Thoughts?

Reply:
I'm happy to maintain the current style, I'll move it below.